File: lesson_8/app/src/main/java/com/kalashnyk/denys/gmapsapp/repository/database/AppDatabase.java
package com.kalashnyk.denys.gmapsapp.repository.database;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

import com.kalashnyk.denys.gmapsapp.repository.database.dao.PinDao;
import com.kalashnyk.denys.gmapsapp.repository.database.entity.PinEntity;

@Database(entities = {PinEntity.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
    public abstract PinDao pinDao();
}

File: lesson_9/app/src/main/java/com/kalashnyk/denys/kotlinbindingsample/repository/database/AppDatabase.kt
package com.kalashnyk.denys.kotlinbindingsample.repository.database

import android.arch.persistence.room.Database
import android.arch.persistence.room.RoomDatabase
import com.kalashnyk.denys.kotlinbindingsample.repository.database.dao.UserDao
import com.kalashnyk.denys.kotlinbindingsample.repository.database.entity.UserEntity

@Database(entities = [(UserEntity::class)], version = 1)
abstract class AppDatabase : RoomDatabase() {
    abstract fun userDao(): UserDao
}